src/library/libraryview.h

```diff
@@ -35,8 +35,9 @@ class LibraryView {
     }
     virtual void saveCurrentViewState() {
     }
-    virtual void restoreCurrentViewState() {
-    }
+    virtual bool restoreCurrentViewState() {
```

Comment:
Please document the meaning of the boolean return value.

Reply:
I copied the description from src/widget/wlibrarytableview.h